pkg/system/chtimes_windows.go

```diff
@@ -24,3 +25,11 @@ func setCTime(path string, ctime time.Time) error {
 	c := windows.NsecToFiletime(windows.TimespecToNsec(ctimespec))
 	return windows.SetFileTime(h, &c, nil, nil)
 }
+
+// setAMTimeNoFollow means to set access/modification time on a file,
+// without following symbol link.
+// The implementation is now a noop on windows, the behavior depends on
+// os.Chtimes
```

Comment:
The description is slightly confusing, as it mentions it's a "no-op" on Windows _and_ that it depends on `os.Chtimes()` - is the "no-op" on the `os.Chtimes()` side, or is the comment outdated?

Based on https://github.com/golang/go/blob/go1.17/src/syscall/syscall_windows.go#L622-L625, I'm guessing the comment is outdated 😅 

But also see my other comment wether or not we should fall back to using `os.Chtimes()` as part of `ChtimesNoFollow`

Reply:
changed
return `ErrNotSupportedPlatform` on windows, ATM.